core/src/main/java/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractor.kt

```diff
@@ -518,7 +508,7 @@ fun zoneOccupationChangeEvents(
             if (currentOffset.distance > envelope.endPos.meters) break
             val entryOffset = Offset.max(Offset.zero(), currentOffset)
             val entryTime =
-                envelope.interpolateTotalTimeUS(entryOffset.distance.meters).microseconds
+                envelope.interpolateDepartureFromUS(entryOffset.distance.meters).microseconds
```

Comment:
```suggestion
                envelope.interpolateArrivalAtUS(entryOffset.distance.meters).microseconds
```

It's the zone *entry* time

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/7609/commits/b7f044ebd08664f02948f7aafb19982fee469f65